var/spack/repos/builtin/packages/py-keras/package.py

```diff
@@ -40,6 +40,6 @@ class PyKeras(PythonPackage):
     version('1.1.0', 'd1711362ac8473238b0d198d2e3a0574')
 
     depends_on('py-setuptools', type='build')
-    depends_on('py-theano', type=('build', 'run'))
+    depends_on('py-theano', type=('build'))
```

Comment:
Are you sure this is only needed at build time? In the tarball I see a lot of `import theano`.